doc/source/conf.py

```diff
@@ -561,7 +561,11 @@ class AccessorCallableDocumenter(AccessorLevelDocumenter, MethodDocumenter):
     priority = 0.5
 
     def format_name(self):
-        return MethodDocumenter.format_name(self).rstrip(".__call__")
```

Comment:
why is it complaining about this?  this just seems more verbose

Reply:
I agree it is more verbose. I the style changes into separate commits so that they can easily be dropped.

`(r)strip` removes an arbitrary set of characters. `strip("_")` is the same as `strip("__")`. Calling `(r)strip` with multiple characters might imply to the reader that exactly this substring is removed, which is not correct.

`".....ccaaa__caller.__caller".rstrip(".__caller__") == ""`